app/templates/search/_briefs_pagination_link.html

```diff
@@ -0,0 +1 @@
+<a href="?page={{ page_number }}{% for key, val_list in link_args.lists() %}{% if key != 'page' %}{% for val in val_list %}&{{ key }}={{ val }}{% endfor %}{% endif %}{% endfor %}">{{ link_label }}</a>
```

Comment:
I think you can simplify this by using url_for.  Something like url_for('main.endpoint', **args).  Have you tried?